src/Umbraco.Cms.Api.Common/DependencyInjection/ProcessRequestContextHandler.cs

```diff
@@ -10,14 +10,15 @@ public class ProcessRequestContextHandler
     : IOpenIddictServerHandler<OpenIddictServerEvents.ProcessRequestContext>, IOpenIddictValidationHandler<OpenIddictValidationEvents.ProcessRequestContext>
 {
     private readonly IHttpContextAccessor _httpContextAccessor;
-    private readonly string _backOfficePathSegment;
+    private readonly string[] _pathsToHandle;
 
     public ProcessRequestContextHandler(IHttpContextAccessor httpContextAccessor)
     {
         _httpContextAccessor = httpContextAccessor;
-        _backOfficePathSegment = Constants.System.DefaultUmbracoPath.TrimStart(Constants.CharArrays.Tilde)
+        var backOfficePathSegment = Constants.System.DefaultUmbracoPath.TrimStart(Constants.CharArrays.Tilde)
             .EnsureStartsWith('/')
             .EnsureEndsWith('/');
+        _pathsToHandle = [backOfficePathSegment, "/.well-known/openid-configuration"];
```

Comment:
I don't feel like we need to store this string in a const/configuration as we should not need it outside of this class.